data/json/construction.json

```
@@ -4016,6 +4017,19 @@
     "pre_terrain": "t_sand",
     "post_special": "done_extract_maybe_revert_to_dirt"
   },
+  {
+    "type": "construction",
+    "id": "constr_extract_sand_mound",
+    "group": "extract_sand",
+  	"//": "Step 2 : extract sand",
```

Two spaces indentation
```suggestion
  	 "//": "Step 2 : extract sand",
```